easybuild/tools/filetools.py

```diff
@@ -944,7 +944,9 @@ def det_patched_files(path=None, txt=None, omit_ab_prefix=False, github=False, f
     patched_regex = re.compile(patched_regex, re.M)
 
     if path is not None:
-        txt = read_file(path)
+        # take into account that file may contain non-UTF-8 characters;
+        # so, read a byyte string, and decode to ascii string (ignoring any non-ascii characters);
+        txt = read_file(path, mode='rb').decode('ascii', 'ignore')
```

Comment:
"`byyte`" is misspelled.
Would `'replace'` be slightly better than `'ignore'` as error mode for `decode`? It would provide slightly more disambiguation in the case of filenames like `"foo"` and `"foo\xa0"`.
Something like `'backslashreplace'` would be more informative, but that's not available in older Python.

Reply:
Good point, fixed in 3f369d6f1